components/dialog/dialog.js

```diff
@@ -71,30 +71,11 @@ class Dialog extends LocalizeStaticMixin(AsyncContainerMixin(DialogMixin(LitElem
 		`];
 	}
 
-	static get resources() {
-		return {
-			'ar': { close: 'إغلاق مربع الحوار هذا' },
-			'da': { close: 'Luk denne dialogboks' },
-			'de': { close: 'Dieses Dialogfeld schließen' },
-			'en': { close: 'Close this dialog' },
-			'es': { close: 'Cerrar este cuadro de diálogo' },
-			'fr': { close: 'Fermer cette boîte de dialogue' },
-			'ja': { close: 'このダイアログを閉じる' },
-			'ko': { close: '이 대화 상자 닫기' },
-			'nl': { close: 'Dit dialoogvenster sluiten' },
-			'pt': { close: 'Fechar esta caixa de diálogo' },
-			'sv': { close: 'Stäng dialogrutan' },
-			'tr': { close: 'Bu iletişim kutusunu kapat' },
-			'tr-tr': { close: 'Bu diyalog kutusunu kapat' },
-			'zh': { close: '关闭此对话框' },
-			'zh-cn': { close: '关闭此对话' },
-			'zh-tw': { close: '關閉此對話方塊' }
-		};
-	}
-
 	constructor() {
 		super();
 		this.width = 600;
+
+		this._namespace = 'components.dialog.dialog';
```

Comment:
Nit: For cases like this where there's only 1-2 terms, I don't think I'd pull it out into a variable -- just reference it directly in the `render()`.